libs/designer/src/lib/core/state/workflow/workflowSelectors.ts

```diff
@@ -180,25 +180,50 @@ export const useNodeGraphId = (nodeId: string): string => {
   );
 };
 
-export const useGetAllAncestors = (nodeId: string) => {
+// BFS search for nodeId
+const getChildrenOfNodeId = (childrenNodes: Set<string>, rootNode?: WorkflowNode, nodeId?: string) => {
```

Comment:
Why is the nodeId optional when the method explicitly asks to get the children of that?